.mailmap

```diff
@@ -1526,6 +1527,7 @@ wookie184 <wookie1840@gmail.com>
 wuyudi <wuyudi119@163.com>
 ylemkimon <ylemkimon@naver.com> <mail@ylem.kim>
 znxftw <vishnu2101@gmail.com>
+zolabar <raszoufine@gmail.com>
```

Comment:
You have added two separate entries to the .mailmap file:
https://github.com/sympy/sympy/blob/4fb6b42dd0941f05178aa570edf9ef61162ce960/.mailmap#L138-L148
I'm going to presume that you want your name and email to be recorded as:
```
Zoufiné Lauer-Baré <raszoufine@gmail.com>
```

Reply:
I'm changing this in #24775. Please comment there if you would like a different name/email.

Reply:
Thank you